src/Paket.Core/ProjectFile.fs

```diff
@@ -639,7 +639,7 @@ type ProjectFile =
                         let c = input.[index]
                         if data.Length = 0 && c = ' '
                         then parseComparison data input (index + 1)
-                        elif data.Length = 2 && c <> ' '
+                        elif data.Length = 2 && c <> ' ' && c <> '\''
```

Comment:
Indeed you're right that is a significantly less naive approach. Pull request amended.